library/extractor/src/main/java/com/google/android/exoplayer2/metadata/id3/Id3Decoder.java

```diff
@@ -455,14 +455,29 @@ private static TextInformationFrame decodeTxxxFrame(ParsableByteArray id3Data, i
     byte[] data = new byte[frameSize - 1];
     id3Data.readBytes(data, 0, frameSize - 1);
 
-    int descriptionEndIndex = indexOfEos(data, 0, encoding);
+    int descriptionEndIndex = indexOfTerminator(data, 0, encoding);
     String description = new String(data, 0, descriptionEndIndex, charset);
 
+    // In ID3v2.4, text information frames can contain multiple values delimited by a null
+    // terminator. Thus, we after each "end of stream" marker we actually need to keep looking
+    // for more data, at least until the index is equal to the data length.
+    ArrayList<String> values = new ArrayList<>();
```

Comment:
nit: move this down a few lines closer to where it's first used (probably L470)

Reply:
Resolved in 0eb56f6.